aiobotocore/waiter.py

```diff
@@ -1,14 +1,30 @@
 import asyncio
 
 # WaiterModel is required for client.py import
+from botocore.exceptions import ClientError
 from botocore.waiter import WaiterModel  # noqa: F401
-from botocore.waiter import Waiter, xform_name, NormalizedOperationMethod, \
-    logger, WaiterError
+from botocore.waiter import Waiter, xform_name, logger, WaiterError
 from botocore.docs.docstring import WaiterDocstring
 from botocore.utils import get_service_module_name
 
 
+class NormalizedOperationMethod(object):
```

Comment:
given that we're now replacing `NormalizedOperationMethod`, we need to add an entry in https://github.com/aio-libs/aiobotocore/blob/master/tests/test_patches.py to ensure that if it changes our test breaks and someone reviews that our replacement is still correct.

Reply:
btw I'm not sure if completely broken is correct, the coroutine would be returned but IIRC the caller was doing a `yield from` so the method would still execute.  That support was added after someone noticed waiters were broken and I think he said it worked after the change...so either botocore changed since we implemented this, or the real issue is that the exception handler was not being applied correctly.

Reply:
now the botocore version of `NormalizedOperationMethod` and the aiobotocore one are unrelated. our changes are localized and the only place where our `NormalizedOperationMethod` version gets instantiated is `aiobotocore/waiter.py`.

Reply:
added the digest as requested.